api/src/main/java/org/ehcache/spi/serialization/Serializer.java

```diff
@@ -26,7 +26,17 @@
 
 /**
  * Interface defining the contract used to transform types in a serial form.
- * Implementations must be thread-safe and must define a constructor accepting a single ClassLoader parameter.
+ * Implementations must be thread-safe and must define either or both of the two constructors:
```

Comment:
This piece of information should be there in some class/interface that the user would directly use, right? The user never really uses/interacts with the `DefaultSerializationProvider` class directly. He only uses the `DefaultSerializerConfiguration` class to configure and of course the `Serializer` interface to do the actual implementation. So I'm not sure if moving it to the `DefaultSerializationProvider` class would be a good idea because the user may never see that.